DokkaRoot.md

```diff
@@ -0,0 +1,197 @@
+# Official Android SDK for [Stream Chat](https://getstream.io/chat/sdk/android/)
+
+> This is the official Android SDK for [Stream Chat](https://getstream.io/chat/sdk/android/), a service for building chat and messaging applications. This library includes both a low-level chat SDK and a set of reusable UI components. Most users start with the UI components, and fall back to the lower level API when they want to customize things.
+
+<br />
+<p align="left">
+  <a href="https://github.com/GetStream/stream-chat-android/actions"><img src="https://github.com/GetStream/stream-chat-android/workflows/Build%20and%20test/badge.svg" /></a>
+  <a href="https://android-arsenal.com/api?level=21"><img alt="API" src="https://img.shields.io/badge/API-21%2B-brightgreen.svg?style=flat"/></a>
+  <a href="https://github.com/GetStream/stream-chat-android/releases"><img src="https://img.shields.io/github/v/release/GetStream/stream-chat-android" /></a>
+</p>
+
+### 🔗 Quick Links
+
+* [Register](https://getstream.io/chat/trial/): Create an account and get an API key for Stream Chat
+
```

Comment:
What do you think about adding the [Maker Account](https://getstream.io/maker-account/) link here too?

> Small teams and individuals can also apply for a [Maker Account](https://getstream.io/maker-account/) that allows you to use Startup Plan for free.